### PR TITLE
I've added the square button #178

### PR DESCRIPTION
**Pull request by Te-Stack**
#178

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -74,6 +74,11 @@ <h2 class="text-primary">Our Vision</h2>
         </div>
       </div>
     </section>
+    <!-- Buttons  -->
+    <div class="but-section">
+      <button class="btn-primary">Join the Community</button>
```
Comment (schmelto) [inline index.html:79]: Can you add links to the different locations?

Comment (Te-Stack) [reply index.html:79]: Yes ..But I was not given any links

Comment (schmelto) [reply index.html:79]: I would suggest using https://github.com/chryz-hub/chryz-hub.github.io/issues/new?assignees=&labels=invite+me+to+the+organization&template=invitation.md&title=Please+invite+me+to+the+GitHub+Community+Organization for the Join the community

File: index.html [hunk 2]
```diff
@@ -74,6 +74,11 @@ <h2 class="text-primary">Our Vision</h2>
...
+      <button class="btn-secondary"><i class="fas fa-code-branch"></i> Create Pull Requests </button>
```
Comment (schmelto) [inline index.html:80]: Probably this one :D https://github.com/chryz-hub/chryz-hub.github.io/pulls

Comment (Te-Stack) [reply index.html:80]: Okay will do that

File: index.html (hunk 1 above)
Comment (Te-Stack) [reply index.html:79]: Okay

File: index.html (hunk 2 above)
Comment (Te-Stack) [reply index.html]: Added the links

**Comment by schmelto** [comment]
You can consider adding the `rel="noopener"` property to the link this will improve the lighthouse score

**Comment by Te-Stack** [comment]
Okay

**Comment by Te-Stack** [comment]
Done that

File: index.html [hunk 3]
```diff
@@ -74,6 +74,11 @@ <h2 class="text-primary">Our Vision</h2>
         </div>
       </div>
     </section>
+    <!-- Buttons  -->
+    <div class="but-section">
+      <a href="https://github.com/chryz-hub/chryz-hub.github.io/issues/new?assignees=&labels=invite+me+to+the+organization&template=invitation.md&title=Please+invite+me+to+the+GitHub+Community+Organization" rel="noopener"><button class="btn-primary">Join the Community</button></a>
+      <a href="https://github.com/chryz-hub/chryz-hub.github.io/pulls"><button class="btn-secondary" rel="noopener"><i class="fas fa-code-branch"></i> Create Pull Requests </button></a>
```
Comment (schmelto) [inline index.html:80]:
```suggestion
      <a href="https://github.com/chryz-hub/chryz-hub.github.io/pulls"><button class="btn-secondary" rel="noopener"><i class="fas fa-code-branch"></i> Create Pull Requests</button></a>
```

**Comment by namangupta1399** [comment]
Good work, @Te-Stack 👍 

Please add container class to the parent div.
Also, instead of adding buttons inside anchor tag. You can remove the buttons and give the class to anchor tags. That will work too.

```
<div class="container but-section">
      <a href="https://github.com/chryz-hub/chryz-hub.github.io/issues/new?assignees=&labels=invite+me+to+the+organization&template=invitation.md&title=Please+invite+me+to+the+GitHub+Community+Organization" class="btn-primary" rel="noopener">Join the Community</a>
      <a href="https://github.com/chryz-hub/chryz-hub.github.io/pulls" class="btn-secondary" rel="noopener"><i class="fas fa-code-branch"></i> Create Pull Request</a>
</div>
```

**Comment by Te-Stack** [comment]
Okay @namangupta1399

**Review by Evavic44 — CHANGES_REQUESTED** [review]
Thank you for being patient with us and making these PR's, kindly work with this pointers given so we can merge it and progress further. 👍🏾

File: src/assets/css/styles.css [hunk 4]
```diff
@@ -93,4 +93,14 @@ nav ul li a:focus {
     background-color: #fff;
     border-radius: 12px;
     box-shadow: 4px 4px 4px 3px var(--shadow);
+}
+
+/* Button  */
+.but-section {
```
[the remark this answers is not in the document]
Comment (Te-Stack) [reply src/assets/css/styles.css:99]: Okay will input the changes

[the remark this answers is not in the document]
Comment (Evavic44) [reply src/assets/css/styles.css]: Will be waiting for your PR! 👍🏾

**Review by Evavic44 — APPROVED** [review]
LGTM! 👍🏾 Any other review guys? so we can merge this and close this issue. #178

**Comment by namangupta1399** [comment]
@Evavic44, should links open in same or new tab?
@Te-Stack , pls change **Create Pull Requests** to **Create Pull Request**. 

Rest looks good. 👍

File: src/assets/css/styles.css (hunk 4 above)
[the remark this answers is not in the document]
Comment (Te-Stack) [reply src/assets/css/styles.css]: Already opened a PR @Evavic44

**Comment by Te-Stack** [comment]
Just waiting to be merged

**Comment by Evavic44** [comment]
> @Evavic44, should links open in same or new tab?
> @Te-Stack , pls change **Create Pull Requests** to **Create Pull Request**.
> 
> Rest looks good. 👍

Or we can just merge it and raise a new `#goodfirstissue` to that.

**Comment by namangupta1399** [comment]
> > @Evavic44, should links open in same or new tab?
> > @Te-Stack , pls change **Create Pull Requests** to **Create Pull Request**.
> > Rest looks good. 👍
> 
> Or we can just merge it and raise a new `#goodfirstissue` for the typo.
> Yes, links should open up a new tab. But the `target` property isn't added as well, we can raise a new issue for both of them

Sound good 👍

**Comment by Evavic44** [comment]
Merged. Thanks for your contribution @Te-Stack 👍🏾

**Pull request merged**